apps/tangle-dapp/data/Payouts/usePayoutsAvailability.ts

```diff
@@ -0,0 +1,32 @@
+import { isSubstrateAddress } from '@webb-tools/dapp-types';
+import { useEffect, useMemo, useState } from 'react';
+
+import useActiveAccountAddress from '../../hooks/useActiveAccountAddress';
+import { convertToSubstrateAddress } from '../../utils';
+import usePayouts from '../DelegationsPayouts/usePayouts';
+
+const usePayoutsAvailability = () => {
+  const activeAccountAddress = useActiveAccountAddress();
+
+  const substrateAddress = useMemo(() => {
+    if (!activeAccountAddress) return '';
```

Comment:
Idea: We should avoid using `''` as a return/fallback/default value, because although it does satisfy the compiler, it may lead to logic bugs on the UI or other parts of the code that depend on this, as they may expect actual addresses rather than an empty string, and thus may fail unexpectedly if they don't have appropriate error handling.